key.ui/src/main/java/de/uka/ilkd/key/gui/nodeviews/MainFrame.java

```diff
@@ -113,4 +113,8 @@ public boolean isShowTacletInfo() {
     public void scrollTo(int y) {
         scrollPane.getVerticalScrollBar().setValue(y);
     }
+
+    public Component getContent() {
```

Comment:
Component is always a SequentView. My suggestion changes the type to remove all the casts.